mmocr/models/textrecog/plugins/common_plugins.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch.nn as nn
+from mmcv.cnn import PLUGIN_LAYERS
+
+
+@PLUGIN_LAYERS.register_module()
+class Maxpool2d(nn.Module):
+    """nn.Maxpool2d()
+    Args:
+        kernel_size (int | tuple): Kernel size for max pooling layer
+        stride (int | tuple): Stride for max pooling layer
+        padding (int | tuple): Padding for pooling layer
```

Comment:
For type hint:
1. `|` -> `or`
2. If applicable, also clarify the element types in the tuple.

That is, `(int | tuple)` -> `(int or tuple(int))`